README.md

```diff
@@ -24,19 +24,30 @@ module "admin_tier" {
   stage                       = "${var.stage}"
   subnets                     = ["${var.subnets}"]
   zone_id                     = "${module.tf_domain.zone_id}"
+  security_groups             = ["${var.security_groups}"]
+  associate_public_ip_address = "${var.associate_public_ip_address}"
 }
 
 ```
 
 This will create a `id`, `fqdn`, `security_group_id`, `role` and `public_ip`.
 
-This module depends on these modules:
+### Module `tf_domain`
+
+Module `tf_admin` requires another module to be used additionally - `tf_domain`
+
+`tf_admin` uses `tf_hostname` to create a DNS record for created host. `tf_hostname` module needs `zone_id` parameter as an input, and this parameter actually is an output from `tf_domain`
+
+That is why `tf_domain` should be implemented in `root` module when we need `tf_admin`
```

Comment:
That is why `tf_domain` should be implemented in `root` TF manifest when we need `tf_admin`